pkg/kubelet/kubelet.go

```diff
@@ -1150,6 +1152,18 @@ func (kl *Kubelet) GetBoundPods() ([]api.BoundPod, error) {
 	return kl.pods, nil
 }
 
+// GetPodFullName provides the first pod that matches namespace and name, or false
+// if no such pod can be found.
+func (kl *Kubelet) GetPodFullName(namespace, name string) (string, bool) {
```

Comment:
We already have a free function of the same name.  Can this be something like GetPodByName(), return *BoundPod, and let the caller call GetPodFullName() ?